interactions/client/bot.py

```diff
@@ -363,6 +365,55 @@ async def change_presence(self, presence: ClientPresence) -> None:
         """
         await self._websocket._update_presence(presence)
 
+    async def get(
+        self, model: T, http_method: Optional[str] = None, **kwargs: Dict[str, Snowflake]
+    ) -> T:
+        r"""
+        A helper function for getting a model from the Discord API.
+
+        :param model: The model to get.
+        :type model: object
```

Comment:
```suggestion
        :type model: :class:`typing.TypeVar`
```

Please have this to be consistent with the type annotation in the argument signature.

Reply:
Removed the TypeVar due to one of your other suggestions.